Maquetado/scss/styles.scss

```diff
@@ -1,3 +1,11 @@
-.title {
-  color: red;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+$nav-height: 90px;
+$title-decoration-height: 90px;
+$title-decoration-width: 100px;
+$tech-logo-size: 100px;
+
+body {
+	font-family: 'Roboto', sans-serif;
```

Comment:
this is not correctly indented, please review the code style described in [this card](https://trello.com/c/pIE8GdGL/21-guia-de-estilos)